MVVM/ViewModel/RoverMode_UIOverlay.cs

```diff
@@ -19,7 +21,18 @@ public partial class RoverMode_UIOverlay : UIOverlay
 
 	public Task ControlModeChangedSubscriber(MqttClasses.ControlMode newMode)
 	{
-		ControlMode = (int)newMode;
+        ControlMode = (int)newMode;
+		if(ControlMode == 1 && MainViewModel.Settings?.Settings.SpeedLimiter.Enabled == true && MainViewModel.Settings?.Settings.SpeedLimiter.MaxSpeed < 1)
+		{
+			SafeModeIndicator.Visible = true;
+			SafeModeIndicator.Text = $"Safe Mode ON - {Mathf.Round(MainViewModel.Settings!.Settings.SpeedLimiter.MaxSpeed * 100.0)}%";//Rounding may seem unnecessary, but without it numbers higher than 80 would be displayed as 79.99999999999999
+		}
+		else
+		{
+            SafeModeIndicator.Visible = false;
```

Comment:
Well we are back to the point where i complain bout spaces
https://youtu.be/SsoOG6ZeyUI?si=tTyN_l_1fSRR9YI2